packages/dotcom-build-images/src/__test__/index.spec.ts

```diff
@@ -0,0 +1,39 @@
+import { PageKitImagesPlugin } from '../index'
+import webpack from 'webpack'
+import path from 'path'
+
+describe('dotcom-build-images', () => {
+  it('build images', async () => {
+    await new Promise((resolve) =>
+      webpack(
+        {
+          mode: 'none',
+          entry: {
+            // adding an example entry as webpack 4 doesnt support an empty object
+            scripts: path.join(__dirname, '/__fixtures__', 'entry-point.js')
```

Comment:
i _think_ an empty array would work but i'm not sure

Reply:
i did try that but no luck. In Webpack 4 you cant have empty strings, arrays or objects 😞